src/lib.rs

```diff
@@ -792,6 +792,30 @@ macro_rules! impl_ranged {
                 <Self as $crate::traits::RangeIsValid>::ASSERT;
                 Self::new_saturating(self.get().saturating_pow(exp))
             }
+
+            /// Wrapping integer addition. Computes `self + rhs`, wrapping around the numeric
+            /// bounds.
+            #[must_use = "this returns the result of the operation, without modifying the original"]
+            #[inline]
+            pub fn wrapping_add(self, rhs: $internal) -> Self {
+                <Self as $crate::traits::RangeIsValid>::ASSERT;
+                let inner = self.get();
+                let range_len = Self::MAX.get().abs_diff(Self::MIN.get()) + 1;
```

Comment:
I believe this can overflow for types like `RangedX<X::MIN, X::MAX>`.

`Self::MAX` can be replaced with `MAX`; likewise for `MIN`. This is because we're inside the `impl` block that has the `const`s defined.

Reply:
Fix: [c6ff5fd](https://github.com/jhpratt/deranged/pull/9/commits/c6ff5fd581d89b52d3e517d9d5e4b036fa705e06)